app/controllers/glossary_controller.rb

```diff
@@ -0,0 +1,4 @@
+class GlossaryController < ApplicationController
+  def show
```

Comment:
My preference is to be explicit with these actions too. I know you could then potentially argue that we shouldn't rely on implicit `render` too, but hey, we are all friends and arguments are just not our style :heart_eyes: :smile: